setup.go

```diff
@@ -160,7 +167,7 @@ func setup(root *cobra.Command) {
 		if err != nil {
 			return commands.NewRuntimeError(err)
 		}
-		cmdCfg, err = cp.Config(app, conf, cfg)
+		cmdCfg, err = cp.Config(app, conf, cfg, skipPrompts())
```

Comment:
I think the simpler change is to just fix the default for the `--yes` argument based on the environment variable here: https://github.com/splunk/qbec/blob/master/setup.go#L136 (replace `false` with `skipPrompts()`). 

This will avoid have to thread an extra variable all over the place, a variable that arguably should be in the background and not have so much prominence.

Reply:
ofcourse. What am I doing!!!